mriqc/interfaces/synthstrip.py

```diff
@@ -0,0 +1,84 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2022 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+"""SynthStrip interface."""
+import os
+from pathlib import Path
+from nipype.interfaces.base import (
+    CommandLine,
+    CommandLineInputSpec,
+    File,
+    TraitedSpec,
+    traits,
+    Undefined,
+)
+
+_fs_home = os.getenv("FREESURFER_HOME", None)
+_default_model_path = Path(_fs_home) / "models" / "synthstrip.1.pt" if _fs_home else Undefined
+
+if _fs_home and not _default_model_path.exists():
+    _default_model_path = Undefined
+
+
+class _SynthStripInputSpec(CommandLineInputSpec):
+    in_file = File(
+        exists=True,
+        mandatory=True,
+        argstr="-i %s",
+        desc="Input image to be brain extracted",
+    )
+    use_gpu = traits.Bool(
+        False, usedefault=True, argstr="-g", desc="Use GPU", nohash=True
+    )
+    model = File(
+        str(_default_model_path),
+        usedefault=True,
+        exists=True,
+        argstr="--model %s",
+        desc="file containing model's weights",
+    )
+    border_mm = traits.Int(
+        1, usedefault=True, argstr="-b %d", desc="Mask border threshold in mm"
+    )
```

Comment:
Will the underlying command line call accept an argument of 0 here? (i.e. would you be able to set no threshold?)

Reply:
Yes, that would be trivial.